app/src/test/java/org/mozilla/tv/firefox/pocket/PocketVideoRepoTest.kt

```diff
@@ -105,12 +103,12 @@ class PocketVideoRepoTest {
     @Test
     fun `GIVEN pocket key is valid WHEN getting a new instance THEN feed state starts inactive`() {
         val repo = PocketVideoRepo.newInstance(videoStore, { true }, isPocketKeyValid = true)
-        assertEquals(FeedState.Inactive, repo.feedState.blockingFirst())
+        repo.feedState.test().assertValue(FeedState.Inactive)
```

Comment:
nit: would it be more idiomatic to have a `feedStateTestObs` member var?

Reply:
It looks like these two tests are creating a different repo and subject than the ones used in the `@Before` block, so that wouldn't work here.  That is a little bit confusing though, I'll add a comment to that effect.

Reply:
It looks like these two tests are creating a different repo and subject than the ones used in the `@Before` block, so that wouldn't work here.  That is a little bit confusing though, I'll add a comment to that effect.